packages/react/src/Avatar/Avatar.tsx

```diff
@@ -56,7 +42,10 @@ export type AvatarProps = {
 } & HTMLAttributes<HTMLSpanElement>
 
 export const Avatar = forwardRef(
-  ({ label, imageSrc, className, color = 'blue', ...restProps }: AvatarProps, ref: ForwardedRef<HTMLSpanElement>) => {
+  (
+    { label, imageSrc, className, color = defaultColor, ...restProps }: AvatarProps,
```

Comment:
Wrong line, but: the component still has a `ams-avatar--purple` class by default, we can get rid of that

Reply:
Good catch, it’s gone. We don’t even need the explicit default value that isn’t an actual value anywhere anyway.

Reply:
Yes, done. To reuse the `avatarColors` array, I need to import it through an elaborate path. Is that okay?